content/js1/sprints/3/prep/index.md

```diff
@@ -8,4 +8,634 @@ backlog= 'Module-JS1'
 backlog_filter= 'Week 3'
 +++
 
+## Pre-requisites
 
+## Learning objectives
+
+- Write an assertion for a function that implements a solution to a given problem
```

Comment:
I worry that we may be slightly over-tying assertions to _functions_ - do we maybe want to also introduce an example of an assertion that isn't related to a function, just to show it can be done?